third_party/blink/renderer/platform/media/web_media_player_impl.cc

```diff
@@ -98,6 +98,15 @@
 #include "third_party/blink/renderer/platform/media/web_media_source_impl.h"
 #include "ui/gfx/geometry/size.h"
 
+// For BUILDFLAG(USE_STARBOARD_MEDIA)
+#include "build/buildflag.h"
+#if BUILDFLAG(IS_COBALT)
```

Comment:
This is good for now, but just FYI I'll be changing this in the near future so that it's a little simpler to use

Reply:
Done